gui-apps/grim-1.4.0: install bash completions with eclass

parona-source opened the pull request:
* Disable bash-completions switch and instead install bash completions
  with bash-completion-r1 eclass.
* This serves to avoid the build system requiring
  app-shells/bash-completion or having to go out of your way to remove
  this requirement from build system itself.

The bash-completions switch was enabled yesterday here https://github.com/gentoo/gentoo/pull/24182 which then introduced this issue https://bugs.gentoo.org/833595.

MatthewGentoo reviewed (commented):
Since the installed files are changing, you should perform a rev bump so that users rebuild the package.

https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html

(also, does the -9999 ebuild need touching too?)

ArsenArsen commented:
besides the above, check whether the fish completions behave well

thanks for getting to this before I did btw <3

parona-source commented:
@MatthewGentoo Modified commit to so that the changes are in 1.4.0-r1 instead also updated 9999 with these changes. Updated both of them eapi 8 as well as while I was at it.

parona-source commented:
@ArsenArsen the fish completions dont require fish like the bash completions required bash-completion. You can check it here https://github.com/emersion/grim/blob/v1.4.0/contrib/completions/meson.build

ArsenArsen commented:
it still tries to query it. it may be best to force it to use the fallback path to ensure consistency

parona-source commented:
@ArsenArsen changed it so that it installs the fish completions with doins. Also double checked the dependencies and updated them.

thesamesam commented:
Thanks both!
